scrapy/http/response/text.py

```diff
@@ -68,6 +69,10 @@ def body_as_unicode(self):
                       ScrapyDeprecationWarning)
         return self.text
 
+    @property
+    def json(self):
+        return json.loads(self.text)
```

Comment:
- It should include a docstring, and you should add `.. autoattribute:: TextResponse.json` to `docs/topics/request-response.rst`.
- It’s not a blocker, but I would consider https://github.com/scrapy/scrapy/pull/4460’s caching approach including https://github.com/scrapy/scrapy/pull/4460#discussion_r401033472

Reply:
Many thanks for the review. Fixed in https://github.com/scrapy/scrapy/pull/4574/commits/b17a007f3a3d48957fc08e51b4cec483f7494af5